WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorView.swift

```diff
@@ -68,14 +69,26 @@ struct ProductSelectorView: View {
                 VStack(spacing: 0) {
                     InfiniteScrollList(isLoading: viewModel.shouldShowScrollIndicator,
                                        loadAction: viewModel.syncNextPage) {
-                        ForEach(viewModel.productRows) { rowViewModel in
+                        ForEach(viewModel.productsSectionViewModels) {
```

Comment:
Not a blocker, just offering an alternative: If we use the title to perform the loop then we do not need to make `ProductsSectionViewModel` conform to `Identifiable`, or declare an `UUID` [here](https://github.com/woocommerce/woocommerce-ios/blob/1d473a57aeb976cfd8941922ac2e3926692fb938/WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift#L7-L8):

```suggestion
                        ForEach(viewModel.productsSectionViewModels, id: \.title) {
```

Reply:
True, done in e5f670131a70859f421a85cd987df525bd2f0c6a